app/controllers/api/v2/template_combinations_controller.rb

```diff
@@ -28,6 +36,14 @@ def create
       def show
       end
 
+      api :PUT, "/template_combinations/:id", N_("Update template combination")
```

Comment:
should this be nested too?  The routes for each verb look rather inconsistent now.